widgets/VersionList.php

```diff
@@ -97,6 +109,10 @@ protected function getRenderData()
             $items = $result;
         }
 
+        if($this->getConfig('sort', 'asc') == 'desc') {
```

Comment:
@filipac Space between `if` and `(`.
```suggestion
        if ($this->getConfig('sort', 'asc') == 'desc') {
```